subprojects/build-init/src/test/groovy/org/gradle/buildinit/plugins/internal/maven/MavenProjectsCreatorSpec.groovy

```diff
@@ -141,4 +141,60 @@ class MavenProjectsCreatorSpec extends Specification {
         then:
         gradleProject.contains("compileOnly group: 'org.gradle', name: 'build-init', version:'1.0.0'")
     }
+
+    def "creates multi module project with same artifactId"() {
```

Comment:
:x: We should probably also add a test case that tests where `project.groupId` == null.